### ISSUE TITLE
Support for rendering mermaid md

### ISSUE THREAD

**Issue by LeskoIam** [redimp/otterwiki#138]
It would be really great if rendering of [mermaid](http://mermaid.js.org/) markdown could be added?

**Comment by redimp** [comment]
I think adding mermaid rendering would be a useful feature. 

A `mermaid` block like this
````
```mermaid
graph TD;
    A-->B;
```
````
would be rendered when viewing the page. Is that what you have in mind?

**Comment by LeskoIam** [comment]
Exactly. A code block, just like gihub supports it.
```mermaid
flowchart
    NO_MM(["No mermaid support"])
    MM(["Mermaid supported"])
    NO_MM --> MM
```

**Comment by redimp** [comment]
Released in [2.6.0](https://github.com/redimp/otterwiki/releases/tag/v2.6.0).
